### ISSUE TITLE
Backfill "mobile_user_lookup" table from Mobile Commons User API

### ISSUE THREAD

**Issue by namimody** [DoSomething/quasar#42]
Waiting on #70

**Comment by sheyd** [comment]
@namimody Updating and moving this to In-Progress, since it's the next step for essentially populating data into #70 created table. This was based on #41, but we closed that out in favor of #70.

**Comment by sheyd** [comment]
This ticket has 2 parts:
- [ ] Figure out how Josh does data scraping from Mobile Commons API to figure out latest list of users.
- [ ] Figure out best batching process to consolidate users.

**Comment by sheyd** [comment]
Version 1 of the script completed. @namimody I think I should create a separate step for the 2nd part above. The current implementation should work for most of our needs. The 2nd part is a great nice to have, and should be what we use for Quasar, but I don't want to get bogged down too much on v.Next unless we need to.

**Comment by sheyd** [comment]
This has been started. Closing out in favor of creating new bug tickets if things come up.
